---
inmenu: 1
layout: page
title: Research
permalink: /research/
---
## Current Projects

### Species Delimitation and Species Tree Inference
I have a long-standing collaboration with [Ziheng Yang](http://abacus.gene.ucl.ac.uk/ziheng/) at the University College London. During the 1990s we worked
on Bayesian methods for phylogenetic inference using DNA sequence data. In the 2000s we worked mostly on methods for estimating
species divergence times and ancestral population sizes using a combination of fossil calibrations and sequence data.
Over the last decade we have focused on the problem of species delimitation and species tree inference
using multi-locus sequence data. Currently, we are incorporating models of introgression into the species
tree inference program [BPP](http://abacus.gene.ucl.ac.uk/software.html) (Bayesian Phylogenetics and Phylogeography).

### Tardigrade Population Genomics
[Tardigrades](https://en.wikipedia.org/wiki/Tardigrade), or "water bears," are one of the most widespread and least studied groups on earth. They are microscopic animals (less than 1 millimeter in length) and are found in many different aquatic and semi-aquatic environments. They have been independently evolving for at least 500 million years having shared a most recent common ancestor with arthropods. Tardigrades are capable of withstanding extreme envionmental conditions and can undergo cryptobiosis, suspending their metabolism and potentially persisting in this dormant state for decades. They are an important part of the soil ecosystem in extreme regions such as antarctica. Tardigrades and nematodes are probably the earliest animal colonizers in new habitats created by volcanic eruptions or retreating glaciers. They are diploid and sexually reproducing (although parthogenesis also occurs) but little is known about their mode of dispersal or population genetic structure.

The first tardigrade genome was sequenced in 2015. I am working with collaborators [Byron Adams](http://lifesciences.byu.edu/Directories/FacultyStaff/ctl/FacultyProfile/mid/5712/NetID/BJA43) (Brigham Young University) and [Carl Johansson](https://www.researchgate.net/profile/Carl_Johansson2) (Fresno City College) to assemble two additional genomes: one for a eutardigrade and the other for a heterotardigrade. We are analyzing next-generation sequence data from pooled samples (hundreds of individuals) for each species. This allows us to identify single nucleotide polymorphisms (SNPs) for subsequence population genetic analyses in addition to obtaining the genome assemblies. We will study the phylogenetic relationship (and divergence times) between species in these two major groups. This research aims to simultaneously delimit species boundaries, determine phylogenetic relationships, and measure population genetic structure. This is a very challenging statistical problem and one role of my lab is to develop methodologies for "deep diversity" studies that allow the hierarchical structure of species, populations and individuals to be discovered using genomic sequencing data alone. Such techniques are essential for new initiatives (such as the [Earth Biogenome Project](https://www.earthbiogenome.org/)) now on the horizon aiming to sequence the genomes of all life on earth. 

### Mitochondrial Heteroplasmy in Bedbugs 
For the most part, animal mtDNA is maternally inherited in a homoplasmic condition. To maintain this,
mechanisms have evolved to prevent the passage of sperm mitochondria into the developing embryo.
However, in recent years departures from this mode of inheritance have been seen, with heteroplasmy
through paternal leakage detected in a variety of species. The replication of sperm mitochondria in the
egg presents far reaching population and evolutionary level consequences, including the disruption of
mito-nuclear interactions which may reduce organismal fitness, and the introduction of additional
haplotypes leading to the erroneous reconstruction of phylogenetic, phylogeographic, and population
genetic histories. Furthermore, the presence of distinct mitochondrial variants within a single cell
provides opportunities for mtDNA recombination, for which convincing evidence has been recently been
presented in a number of species. Understanding the dynamics of heteroplasmy, paternal leakage, and
recombination, while essential for advancing our knowledge of mitochondrial evolution, has largely been
absent, likely due to the absence of a suitable model system. The short generation time, ease of rearing,
and the extent to which heteroplasmy and recombination occurs, therefore sets the bed bug aside as being
an ideal system in which to address long standing questions in mitochondrial inheritance and evolution.

I am collaborating with [Warren Booth](https://www.booth-lab.org/) at the University of Tulsa
and [Coby Schal](https://projects.ncsu.edu/cals/entomology/schal) at North Carolina State University to
study patterns of paternal mitochondrial leakage and evidence for recombination of mitochondrial DNA in the human
bedbug. We will sequence whole mitochondrial genomes from bedbugs produced by controlled genetic crosses and
for population samples to study these processes. Bedbugs are an important human ectoparasite that has probably been
associated with humans for hundreds of thousands of years. Recently, bedbugs have re-emerged as an important
pest worldwide. These studies will improve our understanding of bedbug genetics as well as potentially
providing new insights into the evolutionary forces that shape mitochondrial genomes in many species.
